roles/mig_controller_prereqs/tasks/apply_ocp3_fixes.yml

```diff
@@ -0,0 +1,34 @@
+# Apply misc cluster fixes for velero/restic integration
+
+- name: Apply OCP3 fixes
+  block:
+    - name: Get AWS account information
+      local_action:
+        module: aws_caller_facts
+      register: caller_facts
+  
+    - name: Get instance info
+      local_action:
+        module: ec2_instance_facts
+        region: "{{ ec2_region }}"
+        filters:
+          "tag:Name": "{{ ansible_user }}-origin3-dev"
+          "tag:creator_arn": "{{ caller_facts.arn }}"
+      register: ec2_instance
+
+    - name: Extract public ip
+      set_fact:
+        instance_ip: "{{ (ec2_instance.instances | selectattr('state.name', 'match', '^running$') | list )[0]['public_ip_address'] }}"
+
+    - name: Add OCP EC2 instance host to in memory inventory
+      add_host:
+        name: "{{ instance_ip }}"
+        ansible_user: "{{ ec2_ssh_user }}"
+        ansible_ssh_private_key_file: "{{ ec2_private_key_file }}"
+
+    - name: Set origin pod dir selinux context to match prod installs (restic fix)
+      shell: "chcon -R -t container_file_t /tmp/openshift.local.clusterup/openshift.local.volumes/pods/"
+      become: true
+      ignore_errors: yes
+      delegate_to: "{{ instance_ip }}"
+  when: cluster_version == '3'
```

Comment:
This may need to be changed in future, when we will have two workflows for the OA all-in-one and origin3-dev. Right now, thanks for the fix.

Reply:
I agree, it is technical debt, but first we need to move away from origin3-dev completely. This one is eliminated at that point as we will not need to relabel anything.